core/src/raw/adapters/kv/backend.rs

```diff
@@ -230,6 +230,59 @@ impl<S: Adapter> Accessor for Backend<S> {
 
         Ok((RpList::default(), pager))
     }
+
+    async fn rename(&self, from: &str, to: &str, _: OpRename) -> Result<RpRename> {
```

Comment:
Please also update the `capability` in `info` (so does typed kv).

Reply:
I have added some doc changes in this PR, I know maybe it is not a good habit.  Do you think I need to separate the changes you mentioned and the documentation changes into another PR? If not, all these things will be completed in the current PR.

Reply:
> I have added some doc changes in this PR, I know maybe it is not a good habit. Do you think I need to separate the changes you mentioned and the documentation changes into another PR?

That's OK. I don't like to split PRs for small changes either.